requirements_test_all.txt

```diff
@@ -224,7 +224,7 @@ aioelectricitymaps==0.4.0
 aioemonitor==1.0.5
 
 # homeassistant.components.esphome
-aioesphomeapi==22.0.0
+aioesphomeapi==22.1.0
```

Comment:
Usually we do the lib bump in a separate PR if its not a breaking change

Reply:
This is the only change between version 22.1.0 and version 22.0.0:
https://github.com/esphome/aioesphomeapi/compare/v22.0.0...v22.1.0

But I will prepare a PR to change the version.

Reply:
#111435